frontend/src/components/MeetingRoom/Toolbar/Toolbar.tsx

```diff
@@ -133,15 +133,19 @@ const Toolbar = ({
   // Displays the right panel buttons - any additional buttons to be displayed that aren't in the center of the toolbar.
   const displayRightPanelButtons = (toolbarButton: ReactElement | false, index: number) =>
     index >= centerButtonLimit && index < rightButtonLimit && toolbarButton;
+  // Array of `false` or right panel button ReactElements to display.
+  const rightPanelButtons = toolbarButtons.map(displayRightPanelButtons);
+  // We display the right panel if we have at least one right panel button to display.
+  const displayRightPanel = rightPanelButtons.reduce((sum, next) => sum + Number(!!next), 0) >= 1;
 
   return (
     <div
       ref={toolbarRef}
-      className="absolute bottom-0 left-0 flex h-[80px] w-full items-center bg-darkGray-100 p-4 md:flex-row md:justify-between"
+      className="absolute bottom-0 left-0 flex h-[80px] w-full flex-row items-center justify-between bg-darkGray-100 p-4"
     >
       <div
         ref={timeRoomNameRef}
-        className={`${toolbarButtonsDisplayed <= 1 ? '' : 'mr-3'} flex justify-start overflow-hidden`}
+        className={`${!displayRightPanel ? 'mr-3 ' : ''}${!displayTimeRoomName ? 'hidden ' : ''}flex flex-1 justify-start overflow-hidden`}
```

Comment:
just a suggestion but this `className` is a bit hard to follow with what's going on. is it possible to break it out to a const string and explain with a little comment why we are doing what we are doing?

Reply:
Sure! I'll do it for this and the right panel buttons container, too.